admin/jobs/river/river.go

```diff
@@ -111,12 +114,12 @@ func New(ctx context.Context, dsn string, adm *admin.Service) (jobs.Client, erro
 
 	riverClient, err := river.NewClient(riverpgxv5.New(dbPool), &river.Config{
 		Queues: map[string]river.QueueConfig{
-			river.QueueDefault: {MaxWorkers: 100},
+			river.QueueDefault: {MaxWorkers: 10},
 		},
 		Workers:      workers,
 		PeriodicJobs: periodicJobs,
 		Logger:       logger,
-		JobTimeout:   time.Hour,
+		JobTimeout:   30 * time.Minute,
```

Comment:
Is there a reason for these changes? I believe some of the non-billing jobs have cases where they might run very long

Reply:
During my testing I found that if process is killed during a running river job then the job gets stuck in running mode even after the restart and only fail when on timeout so thought of decreasing it. I checked and it did not seem that it can take much time, but I can revert it.

Reply:
Ah, that sounds like an issue with our graceful shutdowns. Maybe we don't wait for River to stop before stopping the process?

If that doesn't fix it, we may need to think about alternative solutions, either splitting long-running jobs into many small jobs, or seeing if there's a config to retry scheduled jobs earlier than their timeout in case of restarts.

@kaspersjo might have thoughts on this.

This can be handled in a follow up PR